app/grandchallenge/challenges/models.py

```diff
@@ -676,3 +685,284 @@ def delete_challenge_follows(*_, instance: Challenge, **__):
     ).get()
     Follow.objects.filter(object_id=instance.pk, content_type=ct).delete()
+
+
+class ChallengeRequest(models.Model):
+    class ChallengeTypeChoices(models.TextChoices):
+        """Challenge type choices."""
+
+        T1 = "TYPE_1", _("Type 1 - prediction submission")
+        T2 = "TYPE_2", _("Type 2 - algorithm submission")
+
+    class ChallengeRequestStatusChoices(models.TextChoices):
+        ACCEPTED = "ACPT", _("Accepted")
+        REJECTED = "RJCT", _("Rejected")
+        PENDING = "PEND", _("Pending")
+
+    created = models.DateTimeField(auto_now_add=True)
+    status = models.CharField(
+        max_length=4,
+        choices=ChallengeRequestStatusChoices.choices,
+        default=ChallengeRequestStatusChoices.PENDING,
+    )
+    creator = models.ForeignKey(
+        settings.AUTH_USER_MODEL, null=True, on_delete=models.SET_NULL
+    )
+    title = models.CharField(
+        max_length=64,
+        default="",
+        help_text="The name of the planned challenge.",
+    )
+    challenge_short_name = CICharField(
+        max_length=50,
+        help_text=(
+            "Short name that will be used in url, specific css, files etc. should "
+            "the challenge be accepted. No spaces and special characters allowed."
+        ),
+        validators=[
+            validate_nounderscores,
+            validate_slug,
+            validate_short_name,
+        ],
+        unique=True,
+    )
+    abstract = models.TextField(
+        help_text="Provide a summary of the challenge purpose.",
+    )
+    contact_email = models.EmailField(
+        help_text="Please provide an email that our team can use to contact "
+        "you should there be any questions about your request.",
+    )
+    start_date = models.DateField(
+        help_text="Estimated start date for this challenge.",
+    )
+    end_date = models.DateField(
+        help_text="Estimated end date for this challenge.",
+    )
+    organizers = models.TextField(
+        help_text="Provide information about the organizing team (names and affiliations)",
+    )
+    affiliated_event = models.CharField(
+        blank=True,
+        max_length=50,
+        help_text="Is this challenge part of a workshop or conference? If so, which one?",
+    )
+    structured_challenge_submission_form = models.FileField(
+        null=True,
+        blank=True,
+        upload_to=get_pdf_path,
+        storage=public_s3_storage,
+        validators=[
+            ExtensionValidator(allowed_extensions=(".pdf",)),
+            MimeTypeValidator(allowed_types=("application/pdf",)),
+        ],
+    )
+    task_types = models.ManyToManyField(
+        TaskType, blank=True, help_text="What type of task is this challenge?"
+    )
+    modalities = models.ManyToManyField(
+        ImagingModality,
+        blank=True,
+        help_text="What imaging modalities are used in this challenge?",
+    )
+    structures = models.ManyToManyField(
```

Comment:
A lot of these fields seem to be copied from the Challenge models, maybe it is best to define a mixin (eg, like UUIDModel)?